libpod/container.go

```diff
@@ -588,6 +599,50 @@ func (c *Container) PID() (int, error) {
 	return c.state.PID, nil
 }
 
+// ExecSessions retrieves active exec sessions running in the container
+func (c *Container) ExecSessions() ([]string, error) {
+	if !c.locked {
+		c.lock.Lock()
+		defer c.lock.Unlock()
+
+		if err := c.syncContainer(); err != nil {
+			return nil, err
+		}
+	}
+
+	ids := make([]string, 0, len(c.state.ExecSessions))
```

Comment:
Why do you do this rather then just return the ExecSessions? Is this basically creating a snapshot of the execsessions?

Reply:
I think most people are just going to be interested in whether there are active sessions (which are causing the container to not be able to be removed without -f), not the specifics of what's running in the container.